iModelCore/ECDb/Tests/NonPublished/InstanceReaderTest.cpp

```diff
@@ -1,2332 +1,2332 @@
-
-        ASSERT_TRUE(stmt.GetColumnInfo(4).GetDataType().IsPrimitive());
-        ASSERT_EQ(stmt.GetColumnInfo(4).GetDataType().GetPrimitiveType(), PRIMITIVETYPE_DateTime);
-        ASSERT_STRCASEEQ(expected["dtUtc"].asCString(), stmt.GetValueDateTime(4).ToString().c_str());
-
-        ASSERT_TRUE(stmt.GetColumnInfo(5).GetDataType().IsPrimitive());
-        ASSERT_EQ(stmt.GetColumnInfo(5).GetDataType().GetPrimitiveType(), PRIMITIVETYPE_Integer);
-        ASSERT_EQ(expected["i"].asInt(), stmt.GetValueInt64(5));
-
-        ASSERT_TRUE(stmt.GetColumnInfo(6).GetDataType().IsPrimitive());
-        ASSERT_EQ(stmt.GetColumnInfo(6).GetDataType().GetPrimitiveType(), PRIMITIVETYPE_Long);
-        ASSERT_EQ(expected["l"].asInt64(), stmt.GetValueInt64(6));
-
-        ASSERT_TRUE(stmt.GetColumnInfo(7).GetDataType().IsPrimitive());
-        ASSERT_EQ(stmt.GetColumnInfo(7).GetDataType().GetPrimitiveType(), PRIMITIVETYPE_String);
-        ASSERT_STRCASEEQ(expected["s"].asCString(), stmt.GetValueText(7));
-
-        ASSERT_TRUE(stmt.GetColumnInfo(8).GetDataType().IsPrimitive());
-        ASSERT_EQ(stmt.GetColumnInfo(8).GetDataType().GetPrimitiveType(), PRIMITIVETYPE_String);
-        ASSERT_STRCASEEQ(stmt.GetColumnInfo(8).GetProperty()->GetAsPrimitiveProperty()->GetExtendedTypeName().c_str(), "json");
-        ASSERT_STRCASEEQ(expected["p2d"].Stringify().c_str(), stmt.GetValueText(8));
-
-        ASSERT_TRUE(stmt.GetColumnInfo(9).GetDataType().IsPrimitive());
-        ASSERT_EQ(stmt.GetColumnInfo(9).GetDataType().GetPrimitiveType(), PRIMITIVETYPE_String);
-        ASSERT_STRCASEEQ(stmt.GetColumnInfo(9).GetProperty()->GetAsPrimitiveProperty()->GetExtendedTypeName().c_str(), "json");
-        ASSERT_STRCASEEQ(expected["p3d"].Stringify().c_str(), stmt.GetValueText(9));
-
-        ASSERT_TRUE(stmt.GetColumnInfo(10).GetDataType().IsPrimitive());
-        ASSERT_EQ(stmt.GetColumnInfo(10).GetDataType().GetPrimitiveType(), PRIMITIVETYPE_String);
-        ASSERT_STRCASEEQ(stmt.GetColumnInfo(10).GetProperty()->GetAsPrimitiveProperty()->GetExtendedTypeName().c_str(), "json");
-        ASSERT_STRCASEEQ(expected["b_array"].Stringify().c_str(), stmt.GetValueText(10));
-
-        ASSERT_TRUE(stmt.GetColumnInfo(11).GetDataType().IsPrimitive());
-        ASSERT_EQ(stmt.GetColumnInfo(11).GetDataType().GetPrimitiveType(), PRIMITIVETYPE_String);
-        ASSERT_STRCASEEQ(stmt.GetColumnInfo(11).GetProperty()->GetAsPrimitiveProperty()->GetExtendedTypeName().c_str(), "json");
-        ASSERT_STRCASEEQ(expected["bi_array"].Stringify().c_str(), stmt.GetValueText(11));
-
-
-        // BeJsDocument actual;
-        // actual.Parse(stmt.GetValueText(0));
-        // EXPECT_STRCASEEQ(expected.Stringify(StringifyFormat::Indented).c_str(), actual.Stringify(StringifyFormat::Indented).c_str());
-    }
-}
-
-END_ECDBUNITTESTS_NAMESPACE
+/*---------------------------------------------------------------------------------------------

```

Comment:
Github marks this entire file as changed (12332 lines removed, same amount added).
Looks broken to me, have the line endings changed or something similar?